packages/overlay/src/vaadin-overlay-stack-mixin.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * Returns all attached overlays in visual stacking order.
+ * @private
+ */
+const getAttachedInstances = () =>
```

Comment:
`Overlay.__attachedInstances` can be removed now.

Reply:
Done.